spec/controllers/saml_idp_controller_spec.rb

```diff
@@ -112,12 +119,35 @@
       expect(response).to be_ok
     end
 
-    it 'rejects requests from a wrong cert' do
-      delete :logout, params: UriService.params(
-        OneLogin::RubySaml::Logoutrequest.new.create(wrong_cert_settings),
-      ).merge(path_year: path_year)
+    context 'when the cert is not registered' do
+      it 'rejects requests from a wrong cert' do
+        delete :logout, params: UriService.params(
+          OneLogin::RubySaml::Logoutrequest.new.create(wrong_cert_settings),
+        ).merge(path_year: path_year)
 
-      expect(response).to be_bad_request
+        expect(response).to be_bad_request
+      end
+
+      it 'tracks the request' do
+        stub_analytics
+
+        delete :logout, params: UriService.params(
+          OneLogin::RubySaml::Logoutrequest.new.create(wrong_cert_settings),
+        ).merge(path_year: path_year)
+
+        expect(@analytics).to have_logged_event(
+          'Logout Initiated',
+          hash_including(sp_initiated: true, oidc: false, saml_request_valid: false),
+        )
+        expect(@analytics).to have_logged_event(
+          :integration_errors_present,
+          error_details: [:invalid_signature],
+          error_types: [:saml_request_errors],
```

Comment:
in my experience, it's easier to filter for these in CW by having them be individual properties with true/false? So like:
```suggestion
          error_types: { saml_request_errors: true },
```
but I know there are `LIKE` queries we can use with the array form as well

Reply:
hmmm i don't feel super strongly about this, since just having an event for these kinds of errors is going to make searching for them much easier. so am happy to give that structure a go.

Reply:
updated in https://github.com/18F/identity-idp/pull/11615/commits/647b7173e042cc8e226c2534fd6fda2cbbf099ed